Essentials/src/com/earth2me/essentials/commands/Commandtpo.java

```diff
@@ -17,6 +18,8 @@ public Commandtpo() {
 
     @Override
     public void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
+        CompletableFuture<Exception> eFuture = new CompletableFuture<>();
+        eFuture.thenAccept(e -> showError(user.getBase(), e, commandLabel));
```

Comment:
What's the functional difference between using these two lines and just calling `getNewExceptionFuture()`?

Reply:
Nothing, I probably wrote that before I made that method. Fixed!